src/backbone.collectionView.js

```diff
@@ -774,7 +782,7 @@
 				axis : "y",
 				distance : 10,
 				forcePlaceholderSize : true,
-				items : this._isRenderedAsTable() ? "> tbody > tr:not(.not-sortable)" : "> li:not(.not-sortable)",
+				items : this._isRenderedAsTable() ? "> tr:not(.not-sortable)" : "> li:not(.not-sortable)",
```

Comment:
We need that `tbody` in there.. due to automatic wrapping . maybe it is all browsers that add the `tbody` automatically, since this logic clearly expects the `tbody` to be there and I'm sure we would have noticed if this did not work in some browsers. So, assuming this is the case, we can actually just use 

```
"mousedown > li, > tbody > tr > td" : "_listItem_onMousedown",
"dblclick > li, > tbody > tr > td" : "_listItem_onDoubleClick",
```

in the events hash!